cpp/pipelines/add_authority_external_ref.cc

```diff
@@ -1,8 +1,10 @@
 /** \file add_authority_wikidata_ids.cc
  *  \brief functionality to acquire wikidata id corresponding to their gnds
  *  \author andreas-ub
+ *  \author Steven Lolong (steven.lolong@uni-tuebingen.de)
  *
  *  \copyright 2021 Universitätsbibliothek Tübingen.  All rights reserved.
+ *  \copyright 2022 Universitätsbibliothek Tübingen.  All rights reserved.
```

Comment:
There should only be 1 `\copyright` related line. For multiple years, please merge them together like this:
`\copyright 2021-2022 Universitätsbibliothek Tübingen.  All rights reserved.`